src/utils/anonymize.ts

```diff
@@ -0,0 +1,22 @@
+import { requestProps } from '../types';
+
+export const anonymize = (options: requestProps) => {
```

Comment:
@filiphric, isn't this implementation of `anonymize` changing the authorization that should be used when making the HTTP request?

Shouldn't it only hide such info in the GUI and send the correct information when making the request?

Reply:
this is changing the props, not the options that are being sent to the server. I tried my best testing this, but if you find the behavior being incorrect let me know, I’ll make sure to fix that

Reply:
In a project where I'm using your plugin, the request is not happening when hiding credentials.

I'll test with a different API to ensure I'm not doing anything wrong.

I will keep you posted.